tide_webform.install

```diff
@@ -8,6 +8,15 @@
 use Drupal\webform\Entity\Webform;
 use Drupal\block\Entity\Block;
 
+/**
+ * Implements hook_install().
+ */
+function tide_webform_install() {
+  module_load_include('inc', 'tide_core', 'includes/helpers');
+  $config_location = [drupal_get_path('module', 'tide_webform') . '/config/optional'];
```

Comment:
Should it just not be optional?

Reply:
@anthony-malkoun, I tried adding it to the config/install/ folder originally but Drupal complains about replacing a preexisting config entity.